checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,946 @@
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protocol
+    path: checkpoint.quantum.firewall.protocol
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Reject Category
+    path: checkpoint.quantum.firewall.reject_category
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Inbound Packets
+    path: checkpoint.quantum.firewall.server_inbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Outbound Packets
+    path: checkpoint.quantum.firewall.server_outbound_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Severity
+    path: checkpoint.quantum.firewall.severity
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: SmartDefense Profile
+    path: checkpoint.quantum.firewall.smartdefense_profile
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source
+    path: checkpoint.quantum.firewall.src
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source Machine Name
+    path: checkpoint.quantum.firewall.src_machine_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Status
+    path: checkpoint.quantum.firewall.status
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Subject
+    path: checkpoint.quantum.firewall.subject
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source IP Port
+    path: checkpoint.quantum.firewall.src_ip_port
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Destination IP Port
+    path: checkpoint.quantum.firewall.dst_ip_port
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Total Bytes
+    path: checkpoint.quantum.firewall.bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+pipeline:
+  type: pipeline
+  name: Checkpoint Quantum Firewall
+  enabled: true
+  filter:
+    query: "source:checkpoint-quantum-firewall"
+  processors:
+    - type: service-remapper
+      name: Define `product` as the official service of the log
+      enabled: true
+      sources:
+        - product
+    - type: date-remapper
+      name: Define `time` as the official date of the log
+      enabled: true
+      sources:
+        - time
+    - name: Lookup for `severity` to `checkpoint.quantum.firewall.severity` field
```

Comment:
This attribute really looks like datadog's status attribute. Unless there's a really strong reason, would you mind making it so that the lookupTable maps to datadog's standard status attribute values,  remap this attribute to status with a status remapper and then remove the facet that exists for this attribute ?